dace/codegen/targets/cpu.py

```diff
@@ -1042,6 +1042,7 @@ def process_out_memlets(self,
             # Tasklet -> array
             if isinstance(node, nodes.CodeNode):
                 if not uconn:
+                    return
```

Comment:
Assuming this is for debugging purposes to avoid the syntax error below - ensure this is not part of the final PR